src/StreamingDataFrames/streamingdataframes/models/rows.py

```diff
@@ -23,14 +26,20 @@ class Row:
         "leader_epoch",
     )
 
+    # TODO: Maybe include headers here for if/when it's a dict?
+    _copy_map = {
+        "value": lambda self, k: deepcopy(getattr(self, k)),
+        "timestamp": lambda self, k: copy(getattr(self, k)),
```

Comment:
Do we really need to use `getattr` here? 
Looks like `k` is always supposed to be the same as a dictionary key, right?

E.g.
```
"value": lambda self, k: deepcopy(self.value),
```

Reply:
True, good catch...faster =)